grails-app/services/org/broadinstitute/orsp/webservice/ApiService.groovy

```diff
@@ -44,14 +44,19 @@ class ApiService {
                  IssueType.NE.name,
                  IssueType.CONSENT_GROUP.name])
         filterUnique(queryService.findByQueryOptions(options).collect {
-            def link = getTagLib().createLink([controller: it.controller, action: 'show', absolute: true])
+            String link = ""
+            if (it.type == IssueType.CONSENT_GROUP.name) {
```

Comment:
This change is fixing broken links creation, an issue not strictly related to this  story. Please avoid doing this and use another branch and specific story / bug in the future.